apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/ShardEngineImpl.java

```diff
@@ -686,7 +686,7 @@ public MigrateState finishShardProcess(CommandParamInfo paramInfo) {
         shardRecoveryDao.hardDeleteAllShardRecovery();
         // call ANALYZE to optimize main db performance after massive copy/delete/update actions in it
         sourceDataSource.analyzeTables();
-        log.debug("Shard process finished successfully", paramInfo.getShardId(),
+        log.debug("Shard process finished successfully, shard id - {}", paramInfo.getShardId(),
```

Comment:
put spent time into log as well, otherwise 'System.currentTimeMillis() - startAllTables' is not needed.